### PR TITLE
Fix: improve "wrong chain" switching UX

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

I see two problems with the current wrong chain UX:

* The chain switch button in the header is unnecessary – the user doesn't need to switch their chain until they perform a transaction action
* The wrong chain warning in the tx modal is not actionable. It's red and scary but doesn't tell the user what to do to fix the problem.

## What changed
* There was a "Switch to X" button in the header – not anymore
* There was a red warning in the modal – it's now ℹ️ and blue

### CODE REVIEW

**Review by iamacook — COMMENTED** [review]
LGTM but the placement feels a bit off inside the warning.

As we switch the chain, informing the user makes sense to me. Perhaps we could change the colour/icon to be that of the `info` variant? I'd even consider removing the `ChainSwitcher` entirely:

image

@liliiaorlenko, do you have any thoughts?

**Comment by liliiaorlenko** [comment]
> LGTM but the placement feels a bit off inside the warning.
> 
> As we switch the chain, informing the user makes sense to me. Perhaps we could change the colour/icon to be that of the `info` variant? I'd even consider removing the `ChainSwitcher` entirely:
> 
> image
> 
> @liliiaorlenko, do you have any thoughts?

I think this version is clear and not intrusive. If it comes to switching networks our goal is to make experience as seamless as possible. The colour of the warning in yellow perhaps is scary, yes, in blue is more calm and rather informative. If we do automatic chain switch, then we inform user of the interface change and thats it. I can customise the warning so it resemble network switch closer. Maybe text can be improved too: "Currently you wallet is at the wrong chain. When pressing submit we automatically change the network and you can continue on your transaction". I think warning should be more of what is happening right now and how its gonna change after you press SUBMIT. Ill prepare the designs shortly.

**Comment by liliiaorlenko** [comment]
WDYT
https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=5361%3A143633&t=5XEwoU3ALBpHHmAW-1
I designed banner a bit more pleasant and less error like. We gonna use our current connected wallet logo, a bit simpler text with fast recognisable title.

**Comment by katspaugh** [comment]
I think that's a bit... blown out of proportion. I raised this whole topic again because it was already too in-your-face.

Screenshot 2023-05-04 at 14 24 45

**Comment by katspaugh** [comment]
I agree, and this is a good solution. Thanks.

**Review by iamacook — COMMENTED** [review]
LGTM but there is a failing test.

**Review by iamacook — APPROVED** [review]

**Comment by francovenica** [comment]
LGTM

The button is gone and the message is present in the modal
I assume nothing about the switching of the network was changed, but still I checked that it still works fine with MM, Mobile pairing, MM and Trezor.

**Comment by francovenica** [comment]
Note:
The button in the welcome page is still actually useful to change to a network you might have safes in (the sidebar only shows Owned networks of the network you are currently connected to). For MM is easy to change networks, but for mobile pairing or the MM phone app is much easier just to click the "Switch to NETWORK" button and see a prompt in your phone that actually enter into the apps and change the network.
TLDR; can we leave the button for the welcome page?

**Comment by katspaugh** [comment]
Good point, Franco!

**Comment by katspaugh** [comment]
@francovenica to address your concern, I've made the network selector stay on the welcome page. So you can change the app's network to load owned safes on that network.

Also we've added a Connect Wallet button in the sidebar.

**Comment by francovenica** [comment]
Works really nice. Thanks
